lib/iris/tests/integration/test_new_axis.py

```diff
@@ -38,6 +38,15 @@ def test_lazy_data(self):
         self.assertTrue(new_cube.has_lazy_data())
         self.assertEqual(new_cube.shape, (1,) + cube.shape)
 
+    def test_masked_unit_array(self):
```

Comment:
@ajdawson Why is this a unit test and not an integration test? I thought that because it was testing the behaviour of an iris function _with_ a specific numpy data type, this would make the test not isolated to a single function. (I'm not being pedantic, I'm just trying to understand testing).